libc/src/__support/fixed_point/fx_bits.h

```diff
@@ -19,6 +21,74 @@
 
 namespace LIBC_NAMESPACE::fixed_point {
 
+template <typename T> struct FXBits {
+private:
+  using fx_rep = FXRep<T>;
+  using StorageType = typename fx_rep::StorageType;
+
+  StorageType value;
+
+  static_assert(fx_rep::FRACTION_LEN > 0);
+
+  static constexpr size_t FRACTION_OFFSET = 0; // Just for completeness
```

Comment:
Also do we need to annotate `LIBC_INLINE_VAR` for these?

Reply:
we shouldn't need `LIBC_INLINE_VAR` for any constexpr variables. It's a linkage problem and if the number is a compile time constant the linker shouldn't need to touch it as far as I know.